app/forms/webauthn_setup_form.rb

```diff
@@ -64,6 +67,12 @@ def name_is_unique
     @name_taken = true
   end
 
+  def validate_transports
+    return if transports.blank?
+    return if transports.difference(WebauthnConfiguration::VALID_TRANSPORTS).blank?
```

Comment:
if we added the `|| []` above we could drop this first line

and then personal style, I like the `-` method for array/set difference personally
```suggestion
    return if (transports - WebauthnConfiguration::VALID_TRANSPORTS).blank?
```

Reply:
Went a slightly different direction in 988762b99 with the validation here, but used the `-` method as part of changes in 72b1837e4 for the validations at the model.